app/controllers/ConfirmationPageController.scala

```diff
@@ -51,31 +51,37 @@ class ConfirmationPageController @Inject()(override val messagesApi: MessagesApi
     implicit request =>
 
       val result: Future[Result] = for {
-        dates: Option[CashTransactionDates] <- cache.get(request.eori)
+        dates <- cache.get(request.eori)
+        email <- customsDataStoreConnector.getEmail(request.eori)
       } yield {
-        checkDatesAndRedirect(dates)
+        email match {
+          case Right(email) =>
+            checkDatesAndEmailAndRedirect(dates, email.value)
```

Comment:
Better to move this line next to case **Right(email) =>** as whenever possible we should try to keep in a single line for the statements (for classes/objects is different as those have parameters as those can be kept on separate lines for better readability). Cause it is not making a long line we can keep in a single line. 

Same for line no 61

Reply:
This has been addressed.